src/NuGet.Core/NuGet.Build.Tasks/RestoreSettingsUtils.cs

```diff
@@ -17,6 +17,11 @@ public class RestoreSettingsUtils
         public static readonly string Clear = nameof(Clear);
 
         public static ISettings ReadSettings(string solutionDirectory, string restoreDirectory, string restoreConfigFile, Lazy<IMachineWideSettings> machineWideSettings)
```

Comment:
I assume we're keeping this because of Mono?

Reply:
I left this because its  public and I don't want to make any breaking changes